src/styl/_helpers/helpers-cursor.twig

```diff
@@ -0,0 +1,3 @@
+<div class="box">
+    <p class='zoomIn'>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Porro maxime tempora nesciunt ipsa earum?</p>
+</div>
```

Comment:
I don't recall using any `Lorem ipsum` in Colette. Just put `Exemple of paragraph`. Also, for a better demonstartion, put your helper on the container. And maybe a light padding for esthetic?
```suggestion
<div class="box zoomIn pa1">
    <p>Exemple of paragraph</p>
</div>
```

Reply:
👍

Reply:
I'm unresolving this since you didn't followed my reco..

Reply:
oups forget to put lorem after the mention sorry